src/components/FormStep/hooks.ts

```diff
@@ -191,6 +191,14 @@ export const useFormioFormConfigurationParameters = (): Pick<
     [form, submissionId]
   );
 
+  const fetchDigitalAddresses = useCallback(
+    async (componentKey: string) => {
+      const endpoint = `${baseUrl}prefill/plugins/communication-preferences`;
+      return (await get(endpoint, {submissionId, componentKey}))!;
```

Comment:
please implement this like the rest of the api interaction in the `src/data` folder, so that we can add types at that layer and have strongly typed code down the line.